src/container.cpp

```diff
@@ -154,19 +152,19 @@ bool Container::unserializeItemNode(OTB::Loader& loader, const OTB::Node& node,
 void Container::updateItemWeight(int32_t diff)
 {
 	totalWeight += diff;
-	if (Container* parentContainer = getParentContainer()) {
+	if (auto parentContainer = getParentContainer()) {
```

Comment:
Could this be `const auto`
```suggestion
	if (const auto parentContainer = getParentContainer()) {
```